projects/ribbon/evm.js

```diff
@@ -106,6 +112,29 @@ async function avaxTvl(_, _b, { avax: block }) {
   return sumTokens2({ balances, owner: pauserAvax, tokens: [nullAddress, savax], chain, block, transformAddress, })
 }
 
+async function getTreasury(timestamp, block, chainBlocks) {
+  const balances = {};
+  await sumTokensAndLPsSharedOwners(
+    balances,
+    [
+        [RBN, false],
```

Comment:
Remove this line otherwise it looks good. We don't include project native tokens in treasury